crowd_anki/representation/note.py

```diff
@@ -116,12 +118,28 @@ def save_to_collection(self, collection, deck, model_map_cache):
         else:
             self.handle_model_update(collection, model_map_cache)
 
-        self.anki_object.__dict__.update(self.anki_object_dict)
+        self.handle_dictionary_update(note_model)
         self.anki_object.mid = note_model.anki_dict["id"]
         self.anki_object.mod = anki.utils.intTime()
         self.anki_object.flush()
 
         if new_note:
             collection.addNote(self.anki_object)
-        elif not self.config.import_notes_ignore_deck_movement:
+        elif not self.import_config.ignore_deck_movement:
             self.move_cards_to_deck(deck.anki_dict["id"])
+
+    def handle_dictionary_update(self, note_model):
+
+        # Personal Fields Resolution
```

Comment:
extract into a separate function

Reply:
See above.

Reply:
I'd still extract them into separate functions - each time you have a block comment explaining what the given subsection of code does - you'd be be better of extracting it into a separate function with self-explanatory name.  
Really recommend https://simpleprogrammer.com/respecting-abstraction/ as a read on general principles